src/components/Book.js

```diff
@@ -1,20 +1,26 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 
-const Book = ({ book }) => (
+const Book = ({ book, handleRemoveBook }) => (
   <tr>
-    <td>{book.id}</td>
+    <td>{book.id.slice(0, 5)}</td>
```

Comment:
I'm curious, why are you limiting the id to `1-5`?

Reply:
:) hi there bellom. we sliced to 5 char in order to "look better" on the html table for ms 3. but, we will remove this and let the full length of uuid on next mss.
![2020-08-20](https://user-images.githubusercontent.com/18573473/90789109-e2901f80-e2dc-11ea-9725-507e08f8d55d.png)